contracts/DepositLimitModule.vy

```diff
@@ -0,0 +1,123 @@
+# pragma version ~=0.4
+
+"""
+@title Deposit Limit Module for yearn vaults
```

Comment:
You probably want to mention that this module is as temporary measure and will be removed once the vault is battletested